core/tee/tee_svc_cryp.c

```diff
@@ -871,6 +898,121 @@ static void op_attr_value_clear(void *attr)
 	*v = 0;
 }
 
+static TEE_Result op_attr_25519_from_user(void *attr, const void *buffer,
+					  size_t size)
+{
+	uint8_t **key = attr;
+
+	if (size != KEY_SIZE_BYTES_25519 || !*key)
+		return TEE_ERROR_SECURITY;
+
+	memcpy(*key, buffer, size);
+
+	return TEE_SUCCESS;
+}
+
+static TEE_Result op_attr_25519_to_user(void *attr,
+					struct ts_session *sess __unused,
+					void *buffer, uint64_t *size)
+{
+	TEE_Result res;
+	uint8_t **key = attr;
+	uint64_t s = 0;
+	uint64_t key_size = (uint64_t)KEY_SIZE_BYTES_25519;
+
+	res = copy_from_user(&s, size, sizeof(s));
+	if (res != TEE_SUCCESS)
+		return res;
+
+	res = copy_to_user(size, &key_size, sizeof(key_size));
+	if (res != TEE_SUCCESS)
+		return res;
+	if (!key_size)
+		return TEE_SUCCESS;
+
+	if (s < key_size || !buffer)
+		return TEE_ERROR_SHORT_BUFFER;
+
+	return copy_to_user(buffer, *key, key_size);
+}
+
+static TEE_Result op_attr_25519_to_binary(void *attr, void *data,
+					  size_t data_len, size_t *offs)
+{
+	TEE_Result res;
```

Comment:
```c
	TEE_Result res = TEE_ERROR_GENERIC;
	size_t next_offs = 0;

Reply:
`op_attr_25519_to_binary` should we initialize all `*_to_binary` (bignum, value) functions? was here for just consistency

Reply:
We should fix other `*_to_binary()` functions, but that may be the subject for another P-R. As here you introduce new code, prefer have it following optee_os coding style from the start.